streampark-flink/streampark-flink-client/streampark-flink-client-api/src/main/scala/org/apache/streampark/flink/client/bean/DeployResponse.scala

```diff
@@ -17,4 +17,4 @@
 
 package org.apache.streampark.flink.client.bean
 
-case class DeployResponse(address: String, clusterId: String)
+case class DeployResponse(address: String = null, clusterId: String = null, error: Throwable = null)
```

Comment:
I need to make sure if there are any changes related to v2 here

Reply:
> I need to make sure if there are any changes related to v2 here

Only solve the issues in the current branch, it won't affect v2.

Reply:
thanks